test/functional/autoupdate/aup-basic.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME"
+
+	run sudo sh -c "mkdir '$TEST_DIRNAME'/fakepath"
+	# mock systemctl if it doesnt exist to always return true
+	if [ ! -e /usr/bin/systemctl ]; then
+		print "systemctl was not found installed in the system"
+		{
+			printf '#!/bin/bash\n'
+			echo ''
```

Comment:
echo parameters here so you can check if it has run or not.

Reply:
Actually we don't really capture the output of systemctl , so it wont print the parameters.

Reply:
That's true. But we need to find a way to test if the autoupdate is being turned on or not. This test isn't really testing that.

Reply:
For that, we will run this on travis where we install systemd like you said and perform os-install(that will require net access)...but then we can perform real tests.

Reply:
The we can perform real tests.